dashboards-observability/common/constants/shared.ts

```diff
@@ -78,13 +78,13 @@ export enum VIS_CHART_TYPES {
   Pie = 'pie',
   HeatMap = 'heatmap',
   Text = 'text',
-  Gauge = 'gauge',
-  Histogram = 'histogram',
-  TreeMap = 'tree_map',
-  Scatter = 'scatter',
-  LogsView = 'logs_view',
-  Metrics = 'metrics',
-  TableView = 'data_table',
+  // Gauge = 'gauge',
+  // Histogram = 'histogram',
+  // TreeMap = 'tree_map',
+  // Scatter = 'scatter',
+  // LogsView = 'logs_view',
+  // Metrics = 'metrics',
+  // TableView = 'data_table',
```

Comment:
can we remove comments if not needed?

Reply:
+1 yes let's remove these rather than commented out.

Reply:
Hi @ps48 @mengweieric commented as this is not a permanent removing of the feature, we may uncomment it in future. Anyways I will remove it.

Reply:
we can always add back without commenting or uncommenting.